CONTRIBUTION.md

```diff
@@ -0,0 +1,39 @@
+Branching names:
+---
+Each branch must have a name in the following format with a proper prefix and number of Github's ticket it refers to:
+
+```
+prefix/000-branch-name
+``` 
+
+Allowed prefixes:
+* feature
```

Comment:
I'd shorten 
feature -> feat
bugfix -> fix

if clear enough

Reply:
I would leave them not shorten to be consistent and so I do not have to wonder which version to choose (fix vs. bug)